darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -241,6 +242,14 @@ def validation_step(self, val_batch, batch_idx) -> torch.Tensor:
         self._calculate_metrics(output, target, self.val_metrics)
         return loss
 
+    def on_predict_start(self) -> None:
+        # optionally, activate monte carlo dropout for prediction
+        self.set_mc_dropout(active=self.pred_mc_dropout)
```

Comment:
So this overwrite the dropout probability used during the training? So it is possible to have different values during training and inference, is it correct?

Reply:
No the probability is left unchanged. It just means that during prediction (when generating the samples) it applies dropout when `mc_dropout=True`.